### DISCUSSION TITLE
listenLocalConn failed with error: address already in use, packet loss

### DISCUSSION THREAD

**Discussion by Sfssup** [MetaCubeX/mihomo#1170] [General]
time="2024-04-07T20:12:20.671953582+08:00" level=error msg="listenLocalConn failed with error: address already in use, packet loss (rAddr[*net.UDPAddr]=39.165.222.240:8567 lAddr[*net.UDPAddr]=192.168.31.66:8567)"

最近经常出现这，怎么整？

**Comment by chshouyu** [comment]
我的是这样：

```log
[ error ]listenLocalConn failed with error: invalid argument, packet loss (rAddr[*net.UDPAddr]=17.253.114.253:123 lAddr[*net.UDPAddr]=[2408:8214:423:6d00:7d05:9aa5:45ac:421e]:51106)
```

**Comment by XMethues** [comment]
我也遇到这个问题。我是tailscale突然不通了，每次ping 服务器都会报这个错误。我猜测是tailscale的节点。

**Comment by xianren78** [comment]
errorlistenLocalConn failed with error: address already in use, packet loss (rAddr[*net.UDPAddr]=192.168.2.1:53 lAddr[*net.UDPAddr]=192.168.2.143:56046)

**Comment by lslxgh** [comment]
楼主解决了吗？
